### DISCUSSION TITLE
Calculation method of success rate of page display

### DISCUSSION THREAD

**Discussion by Alberter** [apache/skywalking#10066] [Q&A] [answered]
1.v9.1
2.position
![image](https://user-images.githubusercontent.com/29748672/204955065-8da3b99e-552f-45f3-9c8d-1183f3a9791b.png)
3.sla == All status=true question/Minutes in a period(10000*12/31*100)
![image](https://user-images.githubusercontent.com/29748672/204955285-0e466ab3-e841-4e59-90fd-303a9127b0bf.png)
![image](https://user-images.githubusercontent.com/29748672/204955200-331f4b94-9070-4706-82c9-fc49d04c513b.png)
4.The result is that there is no failed URL through URL tracking, but the success rate is 38.71%
![image](https://user-images.githubusercontent.com/29748672/204956197-b9b3e501-80d2-467b-9c4b-b8b8a0350610.png)
5.How to deal with this or a bug?

**Comment by wu-sheng** [comment, marked as answer]
About the metrics in the table, it is just a simple calculation on the website. You could see the graph by clicking the icon on the left of the number.
The real calculation from the backend would ignore the zero, which is in the service's metrics dashboard only.

For the traces not found, I have no idea. Maybe you set some backend sampling? Or database has some issues?

**Reply by Alberter** [reply]
I understand the number of success rate, so there is no question of step 4. Thank you
